src/chapters/classes/declaring-a-class.rst

```diff
@@ -0,0 +1,259 @@
+   of the same type, and it assigns the key/value pairs.
+#. Parameters are passed into the ``constructor`` method rather than the
+   ``class`` declaration.
+
+As an example, let's set up an ``Astronaut`` class to help us store data about
+our animal crew.
+
+Assigning Properties
+^^^^^^^^^^^^^^^^^^^^
+
+Each animal in our crew has a ``name``, ``age``, and ``mass``, and we assign
+these properties in ``constructor`` as follows:
+
+.. sourcecode:: js
+   :linenos:
+
+   class Astronaut {
+      constructor(name, age, mass) {
+         this.name = name,
+         this.age = age,
+         this.mass = mass
+      }
+   }
+
+The ``this`` keyword defines a key/value pair, where the text attached to
+``this`` becomes the key, and the value follows the equal sign (``this.key =
+value``).
+
+``constructor`` uses the three ``this`` statements (``this.name = name``, etc.)
+to achieve the same result as the object declaration
+``let objectName = {name: someString, age: someNumber, mass: someMass}``. Each
+time the ``Astronaut`` class is called, ``constructor`` builds an object with
+the SAME set of keys, but it assigns different values to the keys based on the
+arguments.
+
+.. admonition:: Note
+
+   Each class requires one and only one ``constructor``. Including more than one
+   results in a syntax error. If ``constructor`` is left out of a class
+   declaration, JavaScript adds an empty ``constructor () {}`` automatically.
+
+Calling a New Class Object
+---------------------------
+
+To create an object from a class, we use the keyword ``new``. The syntax is:
+
+::
+
+   let objectName = new ClassName(arguments).
+
+``new`` creates an **instance** (or example) of the class. Each object
+generated shares the same set of keys as every other object made from the
+class.
+
+.. admonition:: Example
+
+   Let's create objects for two of our crew members - Fox and Hippo.
+
+   .. sourcecode:: js
+      :linenos:
+
+      class Astronaut {
+         constructor(name, age, mass){
+            this.name = name,
+            this.age = age,
+            this.mass = mass
+         }
+      }
+
+      let fox = new Astronaut('Fox', 7, 12);
+      let hippo = new Astronaut('Hippo', 25, 1500);
+
+      console.log(typeof hippo, typeof fox);
+
+      console.log(hippo, fox);
+
+   **Console Output**
+
+   ::
+
+      object object
+
+      Astronaut { name: 'Hippo', age: 25, mass: 1500 }
+      Astronaut { name: 'Fox', age: 7, mass: 12 }
+
+In lines 9 and 10, we call the ``Astronaut`` class twice and pass in different
+sets of arguments. This creates the objects ``fox`` and ``hippo``, which is
+confirmed by the output from line 12.
+
+The output from line 14 shows that ``fox`` and ``hippo`` are both the same
+*type* of object (``Astronaut``). The two share the same *keys*, but they have
+different values assigned to those keys.
+
+   Objects created with the same class are NOT the same. They share an overall
```

Comment:
This needs to be said earlier.

Reply:
I considered doing that, but I feel the emphasis works better after the example. 

This point actually gets mentioned a number of times (though not as emphatically) in earlier paragraphs. The final push is to cement the idea with one last, obvious reminder.